### PR TITLE
HTML Media Capture: update Web IDL tests

### PR DESCRIPTION
**Pull request by Honry**
Use ES2015 BTW

### CODE REVIEW

**Comment by Honry** [comment]
@anssiko, PTAL.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/244432791)

# Firefox (nightly) #
Testing web-platform-tests at revision fb83ad06b274f047a8edc6fb24d2ec012eb33d37
Using browser at version BuildID 20170612100241; SourceStamp 27cad9749cddf68e11fdd4e5d73dad84a8f8cf23
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>1 test ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/6272/html-media-capture/idlharness.html">/html-media-capture/idlharness.html</a></summary>


|                                                     Subtest                                                      | Results |                                          Messages                                          |
|------------------------------------------------------------------------------------------------------------------|---------|--------------------------------------------------------------------------------------------|
|                                                                                                                  | OK      |                                                                                            |
| `HTMLInputElement interface: attribute capture`                                                                  | FAIL    | `assert_true: The prototype object must have a property "capture" expected true got false` |
| `HTMLInputElement interface: [object HTMLInputElement] must inherit property "capture" with the proper type (0)` | FAIL    | `assert_inherits: property "capture" not found in prototype chain`                         |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/244432792)

# Sauce (safari) #
Testing web-platform-tests at revision fb83ad06b274f047a8edc6fb24d2ec012eb33d37
Using browser at version 10.0
Starting 10 test iterations

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/244432798)

# Chrome (unstable) #
Testing web-platform-tests at revision fb83ad06b274f047a8edc6fb24d2ec012eb33d37

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/244432799)

# Sauce (MicrosoftEdge) #
Testing web-platform-tests at revision fb83ad06b274f047a8edc6fb24d2ec012eb33d37

**Comment by anssiko** [comment]
LGTM.

**Comment by Honry** [comment]
@dontcallmedom or @tobie, could you help approve this PR? I need people with write access to approve it.

**Comment by leonhsl** [comment]
LGTM

**Review by leonhsl — APPROVED** [review]

**Comment by Honry** [comment]
@alexshalamov, this PR has been reviewed by @anssiko but need someone with write access to approve it, would you please do me a favor?

**Comment by anssiko** [comment]
@Honry, can you assign me as a reviewer via the GH user interface so that I can +1 and unblock the merge.

Now I see:

Merging is blocked
Merging can be performed automatically with one approved review.

File: html-media-capture/idlharness.html
Comment (alexshalamov) [inline html-media-capture/idlharness.html]: nit: change unrelated to PR description.

File: html-media-capture/idlharness.html
Comment (alexshalamov) [inline html-media-capture/idlharness.html]: nit: same here

**Review by alexshalamov — APPROVED** [review]

**Comment by Honry** [comment]
@anssiko, a reviewer should have the write access to wpt to approve PR, only administrators have the  right to grant the write access, maybe @dontcallmedom could help.

**Comment by Honry** [comment]
@alexshalamov, appreciate your review, I will merge it directly.

**Comment by anssiko** [comment]
@Honry, my expectation is you should be able to merge after receiving reviews, no?

Btw. where is the w-p-t review workflow and responsibilities documented? It'd great to know who to contact when there are issues like this.

**Comment by Honry** [comment]
@anssiko, yes, I have the merge right, but w-p-t requires that all pull requests receive at least one approved review from someone with write permissions.(You can refer to [this](https://help.github.com/articles/about-pull-request-reviews/#required-reviews) for more details.)

I only find the [Test Review Policy](http://web-platform-tests.org/reviewing-tests/index.html), and we could join irc #testing (irc.w3.org, port 6665) to contact an appropriate person.

**Comment by anssiko** [comment]
@Honry, thanks, that addresses my concern for now.

**Pull request merged**
